pages/game.tsx

```diff
@@ -0,0 +1,23 @@
+import Head from "next/head";
+import Display from "../components/display/Display";
+import GamePage from "../components/game/GamePage";
+import styles from "../styles/Home.module.css";
+
+export default function Home() {
+  return (
+    <>
+      <Head>
+        <title>Recycool</title>
+        <link rel="icon" href="/favicon.ico" />
+      </Head>
+
+      <main>
+        <GamePage />
+        <form className={styles.display}>
+          <Display value="00:30" style="time" />
+          <Display value="101" style="score" />
+        </form>
```

Comment:
Why is this a form? Does the user need to input data?

Reply:
🙏
No, I'll have to set a countdown in and the points should be counted.